package.json

```diff
@@ -18,14 +18,18 @@
     "build": "turbo run build",
     "clean": "turbo run clean",
     "dev": "turbo run dev",
+    "test": "turbo run test",
```

Comment:
I am unable to run the test while in the root directory due to the following error, but I can run it just fine inside of the primitives's package: 
```js 
@near-lake/primitives:test:     ENOENT: no such file or directory, open '/Users/roshaans/Code/DataPlatform/near-lake-framework-js/packages/near-lake-primitives/dist/test/../../../blocks/61321189.json'
```
I think when you run npm test in the root, it first builds the project then runs the tests from within the `dist` folder. This causes an additional nesting of folders which leaves this [line](https://github.com/near/near-lake-framework-js/blob/bbe3d60e47f5823084650fb8dbc67892cdcce615/packages/near-lake-primitives/test/block.test.ts#L7) unable to find the block needed for testing.

Reply:
Strange, the command still works for me.

`npm test` won't build the project first, that only happens with `npm install`. The tests don't even need to be built, they are run from the typescript files directly.

Can you try `turbo clean` and then `npm test` again?

Reply:
Ah, I see what the problem is now.

If the project _has_ been built, i.e. there is a `dist/` folder including the outputs, then `jest` will pick up the transpiled `test.js` files and attempt to execute them. Because they are a level deeper than the original test files, the path resolution for `blocks/` doesn't work.

I'll update the `jest` config to ignore `dist/`

Reply:
Fixed in the merge from `main` to `dev-1.1` https://github.com/near/near-lake-framework-js/pull/41